.github/workflows/test.yml

```diff
@@ -23,18 +23,17 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
-    - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+    - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
       with:
         go-version: ${{ needs.get-go-version.outputs.go-version }}
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@82d40c283aeb1f2b6595839195e95c2d6a49081b # v5.0.0
+      uses: golangci/golangci-lint-action@a4f60bb28d35aeee14e6880718e0c85ff1882e64 # v6.0.1
       with:
         version: v1.54.2
         args: |
           --verbose
         only-new-issues: false
-        skip-pkg-cache: true
-        skip-build-cache: true
+        skip-cache: true
```

Comment:
This option is unrelated to the deprecated ones, but it is also set in `consul` because we've had problems with the lint cache previously.